.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+# Build the docker image and push it to GitHub Packages
+
+name: Ledger App Builder Publisher
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/publish.yml
+      - Dockerfile
+
+jobs:
+  build:
+    name: Build and push ledger-app-builder image
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+    - name: Clone
+      uses: actions/checkout@v2
+
+    - name: Build and push ledger-app-builder to GitHub Packages
+      uses: docker/build-push-action@v1
+      with:
+        dockerfile: Dockerfile
+        repository: ledgerhq/ledger-app-builder
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+        tag_with_sha: true
```

Comment:
Is `tag_with_sha` useful? If only `latest` is supported, enabling users to download "not the latest" images, using tags, seems incoherent.

Reply:
Indeed, it's not useful for users. However having the history of published images can help reproducing a specific build.

Reply:
Ok, looks good to me, then :)